shadanan/codeanim
# Improve test documentation and configuration

- Update README to demo how to run tests correctly
- Add a `.env` file that configures the PYTHONPATH for VS Code, so that tests can be run directly in VS Code by clicking the play button
- Add the Ruff linter to ensure code complies with formatting guidelines
